routes/index.js

```diff
@@ -0,0 +1,44 @@
+// CONTROLLER
+var express = require('express');
+var router = express.Router();
+var Pirate = require('../models/pirate');
+
+
+// GET
+router.get('/', function(req, res, next){
+  // index view
+  Pirate.find({}, function(err, pirates){
+    if (err) console.log(err);
```

Comment:
You can pass this error to the `next` callback if you want to spit an error message to the page. Otherwise, the error is happening but basically being ignored.